src/js/disability-benefits/526EZ/helpers.jsx

```diff
@@ -117,3 +137,48 @@ export const privateMedicalRecordsIntro = ({ formData }) => {
   );
 };
 
+export function validatePostalCodes(errors, formData) {
```

Comment:
These are okay to stay in here, but if we expect to have a lot of validation functions, we might consider creating a `validations.js` to house these.

Reply:
Good idea.